app/views/katello/api/v2/alternate_content_sources/permissions.rabl

```diff
@@ -0,0 +1,8 @@
+collection @alternate_content_sources
+
+if params.key?(:include_permissions)
```

Comment:
`/katello/api/v2/alternate_content_sources?include_permissions=false` also causes the permissions to be returned.

Reply:
Why is the create permission not included by default?

Reply:
> Why is the create permission not included by default?

It would be an additional meaningless(?) field for others consuming our API. The permission is only needed for the UI. 

Will update this: `if params.key?(:include_permissions)`

Reply:
I meant more like why do you only need "can_create" and not "can_edit" or "can_view" ?

Reply:
`can_edit `and `can_view` are more record specific so they live in the permissions of individual records whereas for create, especially with empty results when there are no ACSs, we still want to know if user has create permissions for ACS in general to show/hide the create button on the UI.